pkg/config/config_template.yaml

```diff
@@ -845,6 +845,28 @@ api_key:
   #   - '*pass*d*'
 
 {{ end -}}
+{{- if .Compliance }}
+#############################################
+## Security Agent Compliance Configuration ##
+#############################################
+## @param compliance_config - custom object - optional
+## Enter specific configuration for continuous compliance checks.
+# compliance_config:
+
+  ## @param enabled - boolean - optional - default: false
+  ## Set to true to enable continuous compliance checks
+  #
+  # enabled: false
+
+  ## @param dir - string - optional - default: /etc/datadog-agent/compliance.d
+  ## Directory path for compliance checks configuration containing enabled benchmarks
+  #
+  # dir: /etc/datadog-agent/compliance.d
+
+  ## @param check_interval - duration - optional - default: 20m
+  ## Check interval
```

Comment:
I think it's the first time we have a _documented_ config option of type "duration-formatted string".

I'm guessing the format is documented in the go docs here (https://golang.org/pkg/time/#ParseDuration)? I think it'd be worth mentioning this here.

Reply:
Good point, should I link to it as?
```
 ## Check interval (see  https://golang.org/pkg/time/#ParseDuration for available options)
```

Reply:
LGTM 👍

Reply:
also, the question I should have asked is whether it makes sense to leave the unit up to the user (makes sense if we expect large ranges/precise granularity on the duration, but not really if we really don't want anyone to ever set anything lower than 1 minute).

Ultimately, depends on the use cases, and if we go with a duration ideally we should check for non-sensical values in code (e.g. `1ns`). Not something that needs to be implemented right now of course.

Reply:
That's a great point, I will double check, but don't think we have code to prevent low check values, I will open a separate PR for that if we don't.